clayui.com/content/docs/css/utilities/c-spacing.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 'C Spacing Utilities'
+---
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+-   [How to read the class](#how-to-read–the–class)
+-   [How to understand the class](#how-to-understand–the–class)
+-   [How to use the class](#how-to-use–the–class)
+
+</div>
+</div>
+
+Utility classes for the spacing in your pages (without the `!important` attribute)
```

Comment:
What is this exactly? A heading? A subheading? If it is a heading, can we mark it up as such? And if it is not a heading, we should append a trailing full-stop.